src/pathwayvis/components/reactionpanel/panelitem/panelitem.component.html

```diff
@@ -45,17 +41,46 @@
         </md-button>
       </md-list-item>
       <div ng-if="$ctrl.bounds" ng-repeat="item in $ctrl.items">
-        <span class="md-caption">{{ $ctrl.itemDisplay(item) }}</span>
         <br>
-        <div>
-          <span class="md-caption">{{ item.lower_bound }}</span>
-          <span class="md-caption">{{ item.upper_bound }}</span>
-       </div>
-        <md-button class="md-icon-button md-secondary" ng-click="$ctrl.onRemoveItem({item: item})">
-          <md-icon md-font-icon="close">close</md-icon>
-        </md-button>
+        <md-list-item ng-click="$ctrl.clickedItemFunction({item: item})">
+          <span style="float: left; font-size: 16px" class="md-caption">{{ item.lower_bound }}</span>
+          <span style="margin-left: 18px; margin-right: 18px;">
+            <= <span class="md-caption">{{ $ctrl.itemDisplay(item) }}</span>
+          <= </span>
+            <span style="float: right; font-size: 16px" class="md-caption">{{ item.upper_bound }}</span>
+            <md-button class="md-icon-button md-secondary" ng-click="$ctrl.onRemoveItem({item: item})" style="float: right">
+              <md-icon md-font-icon="close">close</md-icon>
+            </md-button>
+        </md-list-item>
+        <div style="border:1px solid black; height: 130px; padding: 2%; margin-top: 5%;" ng-show="$ctrl.clickedItem == item.id || ($first && !$ctrl.clickedItem) || $ctrl.items.length == 1">
+          <span class="md-caption" style="display: block; margin-left:5%; margin-top: 5%;">{{ $ctrl.itemDisplay(item) }}</span>
+          <md-list-item ng-if="item">
+            <input style="float: left; width: 70px" type="number" ng-init="$ctrl.lowerbound=item.lower_bound"  ng-model="$ctrl.lowerbound"/>
+            <span style="margin-left: 20px; margin-right: 20px;">
+              <>
```

Comment:
Could you please look for a better divider? I just used it in the mockup. Moritz had an idea about this,